browser-extension/html/tournesol-iframe.html

```diff
@@ -0,0 +1,4 @@
+<iframe id="x-tournesol-iframe-inner"
+        src="https://tournesol.app/login?embed=1"
+        style="width: 100%; height: 99%;">
+</iframe>
```

Comment:
By default the `<iframe>` is wrapped in a `<body>` with a non-zero margin.  
Can be simplified with an explicit `<body>` to remove the arbitrary "99%": 

```suggestion
<body style="margin: 0;">
<iframe id="x-tournesol-iframe-inner"
        src="https://tournesol.app/login?embed=1"
        style="width: 100%; height: 100%; border: none; margin: 0;">
</iframe>
</body>
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/492/commits/4709ab192ca7da1e39a317e5b19f2cc0c46261b9